rotkehlchen/tests/data/mocks/test_coingecko/coins/dai.json

```diff
@@ -0,0 +1 @@
1a0ad7d"},"klay-token":{"decimal_place":18,"contract_address":"0x078db7827a5531359f6cb63f62cfa20183c4f10c"},"xdai":{"decimal_place":18,"contract_address":"0x44fa8e6f47987339850636f88629646662444217"},"hydra":{"decimal_place":null,"contract_address":"abc2cd00700e06922bcf30fe0ad648507113cc56"},"near-protocol":{"decimal_place":18,"contract_address":"6b175474e89094c44da98b954eedeac495271d0f.factory.bridge.near"}},"block_time_in_minutes":0,"hashing_algorithm":null,"categories":["Harmony Ecosystem","Avalanche Ecosystem","Polygon Ecosystem","Moonbeam Ecosystem","Fantom Ecosystem","Near Protocol Ecosystem","Arbitrum Ecosystem","USD Stablecoin","Moonriver Ecosystem","Decentralized Finance (DeFi)","Stablecoins","Gnosis Chain Ecosystem","BNB Chain Ecosystem","Ethereum Ecosystem","Cronos Ecosystem","Arbitrum Nova Ecosystem","Optimism Ecosystem","Metis Ecosystem","Velas Ecosystem"],"public_notice":null,"additional_notices":[],"description":{"en":"MakerDAO has launched Multi-collateral DAI (MCD). This token refers to the new DAI that is collaterized by multiple assets.\r\n"},"links":{"homepage":["https://makerdao.com/","",""],"blockchain_site":["https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f","https://ethplorer.io/address/0x6b175474e89094c44da98b954eedeac495271d0f","https://blockchair.com/ethereum/erc-20/token/0x6b175474e89094c44da98b954eedeac495271d0f","https://arbiscan.io/token/0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1","https://polygonscan.com/token/0x8f3cf7ad23cd3cadbd9735aff958023239c6a063","https://bscscan.com/token/0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3","https://snowtrace.io/token/0xd586e7f844cea2f87f50152665bcbc2c279d8d70","https://avascan.info/blockchain/c/address/0xd586e7f844cea2f87f50152665bcbc2c279d8d70","https://explorer.mainnet.aurora.dev/token/0xe3520349f477a5f6eb06107066048508498a291b/token-transfers","https://moonriver.moonscan.io/token/0x80a16016cc4a2e6a2caca8a4a498b1699ff0f844"],"official_forum_url":["","",""],"chat_url":["","",""],"announcement_url":["",""],"twitter_screen_name":"MakerDAO","facebook_username":"","bitcointalk_thread_identifier":null,"telegram_channel_identifier":"makerdaoOfficial","subreddit_url":"https://www.reddit.com/r/MakerDAO","repos_url":{"github":[],"bitbucket":[]}},"image":{"thumb":"https://assets.coingecko.com/coins/images/9956/thumb/4943.png?1636636734","small":"https://assets.coingecko.com/coins/images/9956/small/4943.png?1636636734","large":"https://assets.coingecko.com/coins/images/9956/large/4943.png?1636636734"},"country_origin":"","genesis_date":null,"contract_address":"0x6b175474e89094c44da98b954eedeac495271d0f","sentiment_votes_up_percentage":66.67,"sentiment_votes_down_percentage":33.33,"market_cap_rank":18,"coingecko_rank":165,"coingecko_score":40.022,"developer_score":0.0,"community_score":41.386,"liquidity_score":69.787,"public_interest_score":0.018,"public_interest_stats":{"alexa_rank":30241,"bing_matches":null},"status_updates":[],"last_updated":"2023-01-20T11:35:21.891Z"}
```

Comment:
As I told you in other files, we use only an absolute minimum from this data. Do not blindly copy all of it. Just copy what we need for the test mock. This way we save less data in the repo's tests